### DISCUSSION TITLE
Anonymous collections instead of subscriptions

### DISCUSSION THREAD

**Discussion by secretmango** [Docile-Alligator/Infinity-For-Reddit#896] [Completed Ideas] [answered]
I would love to use the app **more anonymously**.

### What happens
You have to subscribe to Subreddits or create Multireddits.

All these subscriptions are stored on the Reddit servers, which is good for Desktop-Browser-use, but bad for privacy.

### What should happen
It would be great if you could select, which way you want.
- The unprivate but comfortable way
- The local-stored way

This could work like a Feed-Reader, storing which subreddits to browse locally and loading the content from that file. 
So as far as I understood, the only difference would be, where the Lists are stored.

That list could also be ***auto-syncable*** (auto export and auto import), which would increase decentralized but comfortable use.

----

The integration of such a feature could make the Private mode better, as you could have customized Content, without being logged in. Combined with Orbot or a VPN...

Using an Android Emulator like WayDroid, you could use Infinity-For-Reddit from the Browser without problems, in the really good desktop mode, and replace Browser-Reddit completely.

----
Thank you for your work! I am a Patron for quite a while!

**Comment by candroid-man** [comment]
I second this. It would basically be like how NewPipe handles subscriptions, except you would still be able to create posts and comment normally. This shouldn't even affect usability, as your subscriptions would all display the same way (based on the amount of upvotes and comments)

**Comment by Joao-Felisberto** [comment]
I would really like this feature to be added, and I would love to try to implement it. Another thing that could be added is the hability to follow users. Since this is very close to this issue might be worth mentioning. 

It could also be tied into this broader issue (#399) asking for full offline accounts. 

Also, I would like to take the chance and ask about the projects' views on privacy features. The issue asking to implement the libreddit proxy (#547) was not very well received, so I am curious on this: is the project considering strictly privacy focused features?

This one adds to everyone's experience, unlike libreddit, so in my opinion it should be considered.

**Comment by Docile-Alligator** [comment, marked as answer]
Thank you. Subscribing to subreddits and users, along with creating multireddits are already available in anonymous mode.

**Comment by Wladefant** [comment]
You can also track this #895 discussion for orbot support
